src/components/common/SelectInput.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
```

Comment:
You can import the prop types here directly like:
```suggestion
import { array, func, string } from "prop-types";
```